tests/integration/test_remoted/test_configuration/test_basic_configuration_connection_invalid_protocol.py

```diff
@@ -6,6 +6,8 @@
 import pytest
 
 import wazuh_testing.remote as remote
+import wazuh_testing.api as api
```

Comment:
In this case, I prefer to import just the function instead of the complete module.